packages/ui/src/lib/index.js

```diff
@@ -13,9 +13,10 @@ export { default as RadioButtonGroup } from './radioButton/RadioButtonGroup.svel
 export { default as RadioButtonSolid } from './radioButtonSolid/RadioButtonSolid.svelte';
 export { default as RadioButtonGroupSolid } from './radioButtonSolid/RadioButtonGroupSolid.svelte';
 export { default as Select } from './select/Select.svelte';
+export { default as Spinner } from './spinners/Spinner.svelte'
```

Comment:
```suggestion
export { default as Spinner } from './spinners/Spinner.svelte';
```

Prettier wants a semi-colon here.